docs/source/user_guide/extraction.md

```diff
@@ -1,3 +1,120 @@
 # Extracting IDPs from pipeline derivatives
 
-Work in progress!
+Extraction pipelines ingest a subset of processing pipeline outputs into analysis-ready participant- and/or group-level imaging-derived phenotypes (IDPs) useful for particular downstream analysis (e.g., feature tables, connectivity matrices).
+
+Just like with the BIDS conversion and processing pipelines, Nipoppy uses the {term}`Boutiques framework <Boutiques>` to run extraction pipelines.
+
```

Comment:
Let's also add the implementation peculiarities that make extraction pipelines different that proc_pipes.
- the extraction code may or may not be containerized
- it may require containers that generated the processed output